cylc/flow/cfgspec/globalcfg.py

```diff
@@ -708,11 +708,12 @@
             Conf('hosts', VDR.V_STRING_LIST, ['localhost'])
 
     # Platform Groups
-    with Conf('platform groups'):
+    with Conf('platform groups'):  # noqa: SIM117 (keep same format)
         with Conf('<group>'):
```

Comment:
We could combine these two `with` statements into a single one, however, since all of the others in this file are written individually this would break the pattern somewhat.